integration_test/third_party_apps_test/applications/couchbase/debian_ubuntu/install

```diff
@@ -3,7 +3,7 @@
 set -e
 
 sudo apt-get update
-sudo apt-get install -y gnupg bzip2 wget curl libncurses5
+sudo apt-get install -y gnupg bzip2 wget curl libncurses5-dev
```

Comment:
Are you sure? There doesn't seem to be a `libncurses5-dev` in the noble repos. Noble seems to only come with [`libncurses6`](https://packages.ubuntu.com/noble/libncurses6). Were you trying to use a distro-independent name like `libncurses-dev`?

Note that even on focal, `libncurses5-dev` pulls in `libncurses6` ([`libncurses5-dev`](https://packages.ubuntu.com/focal/libncurses5-dev) -> [`libncurses-dev`](https://packages.ubuntu.com/focal/libncurses-dev) -> `libncurses6`), so could potentially break things…

Reply:
Testing on a noble vm with this command allowed the script to run without erroring, but I agree that using an alternative package could be better/cleaner.

Reply:
I imagine at some point it was out of trying to get Kokoro to work back then, but at this point I'm not sure if its in my memory the explicit reason it was added or if its an extra unnecessary install. I'd be down to remove and re-run the integration test.